dispatch/BESCatalogList.h

```diff
@@ -98,13 +98,17 @@ class BESCatalogList: public BESObj {
 
     friend class BESCatalogListTest;
 
+    BESCatalogList();
+
 public:
     typedef std::map<std::string, BESCatalog *>::iterator catalog_iter;
     typedef std::map<std::string, BESCatalog *>::const_iterator catalog_citer;
 
+    BESCatalogList(const BESCatalogList&) = delete;
+    BESCatalogList& operator=(const BESCatalogList&) = delete;
+
     static BESCatalogList * TheCatalogList();
 
-    BESCatalogList();
     virtual ~BESCatalogList();
```

Comment:
Since BESCatalog is a child of BESObj, the destructor should be _~BESCatalogList() override;_ and not _virtual ..._